packages/contracts-bsc/contracts/common/ReclaimerToken.sol

```diff
@@ -1,31 +1,30 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.6.10;
 
-import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import {ITrueCurrency} from "../interface/ITrueCurrency.sol";
-import {ERC20} from "./ERC20.sol";
+import {IBEP20} from "../interface/IBEP20.sol";
+import {BEP20} from "./BEP20.sol";
 
 /**
  * @title ReclaimerToken
- * @dev ERC20 token which allows owner to reclaim ERC20 tokens
- * or ether sent to this contract
+ * @dev BEP20 token which allows owner to reclaim BEP20 tokens
+ * or bnb sent to this contract
  */
-abstract contract ReclaimerToken is ERC20, ITrueCurrency {
```

Comment:
Let's inherit from `IReclaimerToken.sol` here, so that `reclaimBNB()` and `reclaimToken()` don't accidentally drift away from the interface they're supposed to override.